<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInfo</class>
 <widget class="QWidget" name="EditInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditInfo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageBasicInformation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>730</width>
        <height>693</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Basic Information</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gLayBasicInformation"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSpectralData">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>730</width>
        <height>693</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Spectral Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gLaySpectralData"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSpectralAcquisitionConditions">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Spectral Acquisition Conditions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gLaySpectralAcquisitionConditions"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSampleInformation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Sample Information</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gLaySampleInformation"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageAnnotationsAndTags">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Annotations and Tags</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gLayAnnotationsAndTags"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageOriginate">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Originate</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gLayOriginate"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
